pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -105,5 +105,7 @@ buildPythonPackage rec {
       "x86_64-darwin"
     ];
     maintainers = [ maintainers.sheepforce ];
+    # Tests fail
+    broken = true;
```

Comment:
The PySCF tests are notoriously flaky and most of the time off with some minor numerical deviations, where the test assumes tighter convergence of wave functions than reasonable. I will test which of them fail and would prefer If we could blacklist those, instead of breaking this entire very useful package.